### PR TITLE
Clear debt feature

### PR DESCRIPTION
**Pull request by clementbalestrat**

### CODE REVIEW

File: sections/staking/components/BurnTab/BurnTab.tsx [hunk 1]
```diff
@@ -198,10 +198,17 @@ const BurnTab: React.FC = () => {
 		[amountToBurn, gasPrice, monitorHash, walletAddress, isAppReady]
 	);
 
+	const handleClear = async () => {
+		// setup 1inch tx
```
Comment (andytcf) [inline sections/staking/components/BurnTab/BurnTab.tsx:202]: @clementbalestrat add in

// @TOOD setup 1inch tx

File: sections/staking/components/BurnTab/BurnTab.tsx [hunk 2]
```diff
@@ -235,6 +242,13 @@ const BurnTab: React.FC = () => {
 				inputValue = toBigNumber(amountToBurn);
 				isLocked = false;
 				break;
+			case BurnActionType.CLEAR:
+				onBurnChange(maxBurnAmount.toString());
+				handleSubmit = () => handleClear();
+				inputValue = maxBurnAmount;
+				isLocked = true;
+				canClearDebt = false;
```
Comment (andytcf) [inline sections/staking/components/BurnTab/BurnTab.tsx:250]: @clementbalestrat is this meant to be true? not sure if BurnTiles accept this prop yet

**Review by andytcf — CHANGES_REQUESTED** [review]
good stuff, however, i don't see a change in the package.json 

also left some comments

File: sections/staking/components/BurnTab/BurnTab.tsx (hunk 2 above)
Comment (clementbalestrat) [reply sections/staking/components/BurnTab/BurnTab.tsx:250]: it's false for now as clear debt is still marked as "coming soon"

**Comment by clementbalestrat** [comment]
> good stuff, however, i don't see a change in the package.json
> 
> also left some comments

@andytcf yeah not sure why the package-lock has changes. I did `npm i` only. I'll leave it for now

File: sections/staking/components/StakingInfo/ClearDebtInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const ClearDebtInfo = () => {
+	return <div>clear debt info</div>;
```
Comment (evgenyboxer) [inline sections/staking/components/StakingInfo/ClearDebtInfo.tsx:2]: Needs to be translated yeah?

File: sections/staking/components/helper.ts [hunk 4]
```diff
@@ -45,3 +45,11 @@ export function getTransferableAmountFromBurn(
 		)
 	);
 }
+
+export function sanitiseValue(value: BigNumber) {
```
Comment (evgenyboxer) [inline sections/staking/components/helper.ts:49]: I believe we use American English, so I guess `sanitizeValue`?

**Review by evgenyboxer — APPROVED** [review]
@clementbalestrat LGTM had small comments

File: sections/staking/components/StakingInfo/ClearDebtInfo.tsx (hunk 3 above)
Comment (clementbalestrat) [reply sections/staking/components/StakingInfo/ClearDebtInfo.tsx:2]: Placeholder

File: sections/staking/components/helper.ts (hunk 4 above)
Comment (clementbalestrat) [reply sections/staking/components/helper.ts:49]: Lol who said that

**Pull request merged**